app/src/scripts/admin/admin.store.js

```diff
@@ -192,6 +194,66 @@ let UserStore = Reflux.createStore({
     this.update({ eventLogs, filteredLogs })
   },
 
+  // ** Filters Eventlog to smaller logs based on job status *
+  // uploading contains all jobs
+  filterLogs() {
+    let eventLogs = this.data.eventLogs
+    // let failedLogs = []
+    // let successLogs = []
+    let uploadedLogs = []
+    if (!eventLogs.length) {
+      this.getEventLogs()
```

Comment:
Why is this necessary? I think this action shouldn't result in a call to getEventLogs, a caller should emit both actions if that is expected.